sdk/monitor/monitor-query/src/metricsBatchQueryClient.ts

```diff
@@ -37,29 +44,31 @@ export class MetricsBatchQueryClient {
 
   constructor(tokenCredential: TokenCredential, options?: MetricsBatchQueryClientOptions) {
     let scope;
-    if (options?.batchEndPoint) {
-      scope = `${options?.batchEndPoint}/.default`;
+    if (options?.batchMetricsScope) {
+      scope = `${options?.batchMetricsScope}/.default`;
     }
     const credentialOptions = {
       credentialScopes: scope,
     };
     const packageDetails = `azsdk-js-monitor-query/${SDK_VERSION}`;
+    const batchEndPoint = options?.batchEndPoint ?? "https://westus2.metrics.monitor.azure.com/";
```

Comment:
Endpoint is required and should be set by the customer. We should not use a default value when it's not set.

Reply:
Done